sentry_sdk/tracing_utils.py

```diff
@@ -376,6 +377,24 @@ def serialize(self, include_third_party=False):
         return ",".join(items)
 
 
+def should_propagate_trace(url, trace_propagation_targets):
+    # type: (str, List[str]) -> bool
+    """
+    Returns True if trace_propagation_targets are not set, or if the url matches any of the set targets. Otherwise, returns False.
+    """
+    if trace_propagation_targets is None or trace_propagation_targets == []:
```

Comment:
can we define semantics more precisely, if `None` is the default that's fine and you can treat it as `allow all` but empty array should be empty array. Having both `None` and `[]` take the same semantics is not clean here.

Reply:
Empty array is the default and means: send to everywhere.
The check for None is just a bit paranoid so in case that someone sets `sentry_sdk.init(trace_propagation_targets=None)` nothing breaks.

Reply:
I have now made the code better readable.